### DISCUSSION TITLE
Mamba f405 softserial help

### DISCUSSION THREAD

**Discussion by surfshag** [iNavFlight/inav#6691] [Q&A]
HI All.

Can anyone help?
I have upgraded from 2.5 to 2.6.1 with a full chip wipe.
I have a Mamba F405 (uk) flight controller with Crossfire and GPS using to UARTS so had my smart audio setup as softserail on the PPM pad.
After the upgrade this has stopped working.
For the life of me i dont remember how I got it to work on inav 2.5
I have soft serial enababled and smartaudio selected but its not working.

Am I doing something wrong or does softserarial need mapping to the PPM pad some how?

Thanks for any help and go steady with me this is my first post.

Cheers 

Gary

**Comment by julianschweizer** [comment]
I have Mamba F405 MK2:

resource PPM 1 NONE
resource ESCSERIAL 1 NONE
resource SERIAL_TX 11 C09
feature SOFTSERIAL
serial 2 2 115200 57600 0 115200
serial 30 2048 115200 57600 0 115200
save
